src/org/jgroups/protocols/JDBC_PING.java

```diff
@@ -112,14 +112,6 @@ public void init() throws Exception {
     }
 
 
-    @Override
-    public void stop() {
-        super.stop();
-        if(is_coord)
-            removeAll(cluster_name);
```

Comment:
Hi @belaban with the change in https://github.com/belaban/JGroups/commit/9b5aeb5ea28d99fb104dafe2e98e6ee9146b5eb5#diff-6b7306e1d2f7cbc9fddffed0e74b452cR134 this code is unreachable and that's why I am removing it.